### PR TITLE
Add missing PSDCS in bridge cache

### PR DESCRIPTION
**Pull request by blegat**
Let's tag MOI v0.6.1 once this is merged since this is a critical fix now that https://github.com/JuliaOpt/JuMP.jl/pull/1468 is merged
Closes https://github.com/JuliaOpt/JuMP.jl/issues/1476

### CODE REVIEW

File: test/bridge.jl [hunk 1]
```diff
@@ -162,6 +162,7 @@ MOIU.@model(NoRSOCModel,
 
     @testset "Combining two briges" begin
         fullbridgedmock = MOIB.fullbridgeoptimizer(mock, Float64)
+        @test MOI.supports_constraint(fullbridgedmock, MOI.VectorAffineFunction{Float64}, MOI.PositiveSemidefiniteConeSquare)
```
Comment (mlubin) [inline test/bridge.jl]: This test doesn't fall under the "combining two bridges" header. Consider using a different testset.

File: test/bridge.jl [hunk 2]
```diff
@@ -160,6 +160,33 @@ MOIU.@model(NoRSOCModel,
...
+    @testset "Supports" begin
+        fullbridgedmock = MOIB.fullbridgeoptimizer(mock, Float64)
+        for F in [MOI.SingleVariable, MOI.ScalarAffineFunction{Float64},
+                  MOI.ScalarQuadraticFunction{Float64}]
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.Interval{Float64})
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64},
+                  MOI.VectorQuadraticFunction{Float64}]
+            @show F
```
Comment (mlubin) [inline test/bridge.jl]: Debug output?

File: test/bridge.jl [hunk 3]
```diff
@@ -160,6 +160,33 @@ MOIU.@model(NoRSOCModel,
...
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.PositiveSemidefiniteConeSquare)
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.GeometricMeanCone)
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}]
+            # Missing vcat for quadratic for supporting quadratic
```
Comment (mlubin) [inline test/bridge.jl]: I don't understand these two comments. Are they TODOs?

Comment (blegat) [reply test/bridge.jl]: Yes :)

File: test/bridge.jl [hunk 4]
```diff
@@ -160,6 +160,32 @@ MOIU.@model(NoRSOCModel,
...
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64},
+                  MOI.VectorQuadraticFunction{Float64}]
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.PositiveSemidefiniteConeSquare)
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.GeometricMeanCone)
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}]
+            # TODO Missing vcat for quadratic for supporting quadratic
```
Comment (mlubin) [inline test/bridge.jl]: I still don't understand these TODOs and how they relate to the surrounding code. Are they implying that certain constraints are not yet supported? Which?

Comment (blegat) [reply test/bridge.jl]: These tests are not in the previous `for` because they do not support VectorQuadraticFunction yet

Comment (mlubin) [reply test/bridge.jl]: Could you clarify that on the comments?

**Pull request merged**